### PR TITLE
feat(response-log): add logging to the response for stripe compatibility layer

### PR DESCRIPTION
**Pull request by ShankarSinghC**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

   This PR enhances the Stripe compatibility as it logs response in the `compatibility_api_wrap`. 

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## How did you test it?

Screenshot 2023-06-19 at 11 53 27 AM

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: crates/router/src/compatibility/wrap.rs [hunk 1]
```diff
@@ -88,10 +96,17 @@ where
...
-        Err(error) => {
-            logger::error!(api_response_error=?error);
-            api::log_and_return_error_response(error)
-        }
-    }
+    let response_code = res.status().as_u16();
+    let end_instant = Instant::now();
+    let request_duration = end_instant.saturating_duration_since(start_instant);
+    logger::info!(
+        tag = ?Tag::EndRequest,
```
Comment (pixincreate) [inline crates/router/src/compatibility/wrap.rs:106]:
```suggestion
        tag =? Tag::EndRequest,
```

File: crates/router/src/connector/airwallex/transformers.rs
Comment (lsampras) [inline crates/router/src/connector/airwallex/transformers.rs:1]: Did you mean to stage this file?
Doesn't seem to be related to other changes...

**Review by lsampras — COMMENTED** [review]
LGTM for the most part,
Can you add a PR description as well

File: crates/router/src/compatibility/wrap.rs (hunk 1 above)
Comment (ShankarSinghC) [reply crates/router/src/compatibility/wrap.rs:106]: Is this a necessary change?

Comment (pixincreate) [reply crates/router/src/compatibility/wrap.rs:106]: Feel free to ignore

**Review by Abhicodes-crypto — APPROVED** [review]

**Review by lsampras — APPROVED** [review]
